WordPressVIPMinimum/ruleset.xml

```diff
@@ -157,10 +157,35 @@
 		<message>`%1$s()` performs a no-LIMIT query by default, make sure to set a reasonable `posts_per_page`. `%1$s()` will do a -1 query by default, a maximum of 100 should be used.</message>
 	</rule>
 
-	<rule ref="WordPressVIPMinimum.Variables.VariableAnalysis">
-		<properties>
-			<property name="allowUnusedCaughtExceptions" value="1"/>
-		</properties>
+	<!-- Include VariableAnalysis checks -->
+	<rule ref="VariableAnalysis"/>
+
+	<!-- Prevent deprecation notice when the sniff is not explicitly included. -->
+	<rule ref="WordPressVIPMinimum.Variables.VariableAnalysis.DeprecatedSniff">
+		<severity>0</severity>
+	</rule>
+
+	<!-- Prevent duplicate messages from deprecated sniff. -->
```

Comment:
I'm a bit confused, how come we want to silence these below warnings if they are being removed in the VariableAnalysisSniff?

Reply:
This is how deprecations are done in WPCS. If I comment these out, I do get duplicate messages, as PHPCS has registered the same violations under two different sniff classes (albeit that both ultimately use the same logic from the new package class).